src/java.base/share/classes/java/lang/String.java

```diff
@@ -2607,6 +2607,19 @@ public int indexOf(String str) {
      * }</pre>
      * If no such value of {@code k} exists, then {@code -1} is returned.
      *
+     * @apiNote
+     * Unlike {@link #substring(int)}, for example, this method does not throw
+     * an exception when {@code fromIndex} is outside the valid range.
+     * Rather, it returns -1 when {@code fromIndex} is larger than the length of
+     * the string.
+     * This result is, by itself, indistinguishable from a genuine absence of
+     * {@code str} in the string.
+     * If stricter behavior is needed, {@link #indexOf(String, int, int)}
+     * should be considered instead.
+     * On {@link String}s {@code s} and a non-empty {@code str}, for example,
```

Comment:
Is both the normal font "s" and the code font "s" needed.
I reads ok as "On String s and a non-empty str,"...